api/include/opentelemetry/trace/span.h

```diff
@@ -22,17 +28,21 @@ enum class SpanKind
   kConsumer,
 };
 /**
- * StartSpanOptions provides options to set properties of a Span at the time of its creation
+ * StartSpanOptions provides options to set properties of a Span at the time of
+ * its creation
  */
 struct StartSpanOptions
 {
   // Optionally sets the start time of a Span.
   //
-  // If the start time of a Span is set, timestamps from both the system clock and steady clock
+  // If the start time of a Span is set, timestamps from both the system clock
+  // and steady clock
   // must be provided.
   //
-  // Timestamps from the steady clock can be used to most accurately measure a Span's
-  // duration, while timestamps from the system clock can be used to most accurately place a Span's
+  // Timestamps from the steady clock can be used to most accurately measure a
+  // Span's
+  // duration, while timestamps from the system clock can be used to most
+  // accurately place a Span's
```

Comment:
Please format here and other places in this file.

Reply:
Formatted.